### DISCUSSION TITLE
Invalid configuration configuring domain

### DISCUSSION THREAD

**Discussion by sostenesapollo** [vercel/community#2218] [Help]
### Summary

I've configured my nameservers in aws route 53, and I'm not getting to set the 
<img width="489" alt="image" src="https://user-images.githubusercontent.com/35838667/234993444-8921996d-da59-4edb-a301-1efd05cc1479.png">
in aws, or better I'm able to set, but I can't access the site, and the vercel site always say the same:

`Invalid configuration`, can you help me ?

ps: My Nameservers are correctly configured in aws, because I'm already able to create subdomains with it and use...
just the base domain that I cant point to vercel.

**Comment by amyegan** [comment]
Hi @sostenesapollo! Are you able to tell me more about the steps you took? When you say you can create subdomains, I'm not sure if you are creating those on AWS or on Vercel.

If you can share you domain here, I would be happy to take a closer look. Otherwise, you may find some of our domain troubleshooting docs helpful:

- [Why is my Vercel domain not verified?](https://vercel.com/guides/why-is-my-vercel-domain-unverified)
- [DNS propagation](https://vercel.com/docs/concepts/projects/domains#dns-propagation)
- [Troubleshooting domains](https://vercel.com/docs/concepts/projects/domains/troubleshooting#)
